### PR TITLE
Add empty signature option - Closes #64670

### LINKED ISSUE #6470: Lisk Elements doesn't calculate the minFee of a multisignature transaction correctly
### Description
The signatures of a given accounts with total members (`optionalKeys.length + mandatoryKeys.length`) more than the required number of signature (`numberOfSignatures`) is an array like:
```
signatures: ['valid_signature_1', 'valid_signature_2', '', 'valid_signature_3']
```
where there are 4 members and `numberOfSignatures` equals to 3.

To get the `minFee` we do:
```
const minFee = transactions.computeMinFee(schema, {
    ...transactionWithoutFee,
    signatures: undefined,
  }, {
    baseFees: BASE_FEES,
    numberOfSignatures,
  });
```
Lisk Elements 5.1.0 doesn't take the empty string placeholder (for the third member who hasn't signed the tx) into account for calculating the transaction byte size, therefore the `minFee` calculated using said byte size is lower than Lisk Core expects. 

### Steps to reproduce
Try broadcasting a multisignature transaction of any type other than `4:0` with fee calculated by `transactions.computeMinFee` where the account has 1 mandatory member, 4 optional members and `numberOfSignatures` is 2.
The broadcast will fail because `fee` is lower than `minFee` required.

### Which version(s) does this affect? (Environment, OS, etc...)
Lisk Elements 5.1.0

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR resolves #6470 

### How was it solved?

- Add `numberOfEmptySignatures` option to specify number of empty signatures to include in calculation

### How was it tested?

- Add test case where signatures need to include empty signature

### CODE REVIEW

File: elements/lisk-transactions/src/fee.ts [hunk 1]
```diff
@@ -22,9 +22,10 @@ interface BaseFee {
 }
 
 interface Options {
-	readonly minFeePerByte: number;
-	readonly baseFees: BaseFee[];
-	readonly numberOfSignatures: number;
+	readonly minFeePerByte?: number;
+	readonly baseFees?: BaseFee[];
+	readonly numberOfSignatures?: number;
+	readonly numberOfEmptySignatures?: number;
```
Comment (ManuGowda) [inline elements/lisk-transactions/src/fee.ts:28]: do we need this extra param? from signatures array we can know the empty buffer and take the count?

Comment (shuse2) [reply elements/lisk-transactions/src/fee.ts:28]: we don't use `signatures` param in this function, so we need to know how many is required and how many signatures are empty

**Review by ManuGowda — APPROVED** [review]

File: elements/lisk-transactions/src/fee.ts [hunk 2]
```diff
@@ -37,14 +38,18 @@ const computeTransactionMinFee = (
 	trx: Record<string, unknown>,
 	options?: Options,
 ): bigint => {
+	const mockSignatures = new Array(
```
Comment (mitsuaki-u) [inline elements/lisk-transactions/src/fee.ts:41]: Is it mock? maybe name it `emptySignatures`?

File: elements/lisk-transactions/src/fee.ts [hunk 3]
```diff
@@ -37,14 +38,18 @@ const computeTransactionMinFee = (
...
 	const size = getBytes(assetSchema, {
 		...trx,
-		signatures: new Array(options?.numberOfSignatures ?? DEFAULT_NUMBER_OF_SIGNATURES).fill(
-			Buffer.alloc(DEFAULT_SIGNATURE_BYTE_SIZE),
-		),
+		signatures: mockSignatures,
 	}).length;
 	const baseFee =
-		options?.baseFees.find(bf => bf.moduleID === trx.moduleID && bf.assetID === trx.assetID)
+		options?.baseFees?.find(bf => bf.moduleID === trx.moduleID && bf.assetID === trx.assetID)
```
Comment (mitsuaki-u) [inline elements/lisk-transactions/src/fee.ts:52]:
```suggestion
		options?.baseFees?.find(baseFee => baseFee.moduleID === trx.moduleID && baseFee.assetID === trx.assetID)
```

**Review by mitsuaki-u — APPROVED** [review]
LGTM

File: elements/lisk-transactions/src/fee.ts (hunk 2 above)
Comment (shuse2) [reply elements/lisk-transactions/src/fee.ts:41]: it's both 64bytes and 0 bytes, so empty is not correct I think

Comment (ManuGowda) [reply elements/lisk-transactions/src/fee.ts:41]: dummySignatures?
